### PR TITLE
std/time: Give an example to get UNIX_EPOCH in seconds

### PR DESCRIPTION
**Pull request by kallisti5**
A tiny doc improvement.  I always end up looking for EPOCH and always find obtaining it less-than-obvious from the docs.

### CODE REVIEW

File: src/libstd/time/mod.rs [hunk 1]
```diff
@@ -382,6 +382,15 @@ impl fmt::Debug for SystemTime {
 /// [`SystemTime`] instance to represent another fixed point in time.
 ///
 /// [`SystemTime`]: ../../std/time/struct.SystemTime.html
+///
+/// # Examples
+///
+/// ```no_run
+/// use std::time::{SystemTime,UNIX_EPOCH};
```
Comment (GuillaumeGomez) [inline src/libstd/time/mod.rs]: Use missing whitespace after comma.

File: src/libstd/time/mod.rs [hunk 2]
```diff
@@ -382,6 +382,15 @@ impl fmt::Debug for SystemTime {
...
+///
+/// let now = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs();
```
Comment (GuillaumeGomez) [inline src/libstd/time/mod.rs]: Can please not use `unwrap`? We try to avoid it in the docs.

**Comment by kallisti5** [comment]
Done.  I know unwraps are generally not great with docs.. but if the system time is before UNIX_EPOCH (which shouldn't even be possible in most modern BIOS's, a panic is well deserved.)

Adjusted to show the panic occurring.

File: src/libstd/time/mod.rs [hunk 3]
```diff
@@ -382,6 +382,18 @@ impl fmt::Debug for SystemTime {
 /// [`SystemTime`] instance to represent another fixed point in time.
 ///
 /// [`SystemTime`]: ../../std/time/struct.SystemTime.html
+///
+/// # Examples
+///
+/// ```no_run
+/// use std::time::{SystemTime, UNIX_EPOCH};
```
Comment (GuillaumeGomez) [inline src/libstd/time/mod.rs]: You could just rewrite it as follows:

```
match SystemTime::now().duration_since(UNIX_EPOCH) {
    Ok(n) => println!("1970-01-01 00:00:00 UTC was {} seconds ago!", now.as_secs()),
    Err(_) => panic!("SystemTime before UNIX EPOCH!"),
}
```

**Comment by GuillaumeGomez** [comment]
Once updated, please [squash](https://blog.guillaume-gomez.fr/Git-tips/1/2) your commits.

**Comment by kallisti5** [comment]
Well, since squashing involves rebasing and you can't rewrite remote histories... i'm just going to open another PR :-|

**Comment by Mark-Simulacrum** [comment]
It's fine to squash and push to the PR branch. At least for Rust's repositories, PR branches are generally not considered "public" in that others' have pulled them and as such can be freely rewritten and force pushed at any time.

**Pull request closed without merging**